bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ metadata:
     containerImage: quay.io/opentelemetry/opentelemetry-operator
     createdAt: "2020-12-16T13:37:00+00:00"
     description: Provides the OpenTelemetry components, including the Collector
-    operators.operatorframework.io/builder: operator-sdk-v1.8.0+git
```

Comment:
Current version was also generated with 1.9 but the build was tagged 1.8.0+git wrongly